learn/tanssi/index.md

```diff
@@ -0,0 +1,9 @@
+---
+title: Learn About Tanssi
+description: Learn all about Tanssi, how to build your appchain with the rust language based substrate framework and how easy is to deploy your Appchain in the polkadot web3 ecosystem
```

Comment:
Try to avoid having duplicate titles and descriptions - this is the exact same as the `learn/index.md` page